_src/_layout.jade

```diff
@@ -1,6 +1,8 @@
 doctype
 html
   head
-    link(rel="stylesheet" href="/main.css")
+    link(rel="stylesheet" href="css/main.css")
+    link(href="https://fonts.googleapis.com/css2?family=Oswald:wght@200&display=swap" rel="stylesheet")
+    link(href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet")
```

Comment:
@felipprodrigues you can merge both links into one to avoid too many requests. Take a look on Camelias or my website code.